__init__.py

```diff
@@ -1,6 +1,6 @@
 # File: __init__.py
 #
-# Copyright (c) 2019-2021 Splunk Inc.
+# Copyright (c) 2019-2022 Splunk Inc.
```

Comment:
btw you don't need to manually update all of these anymore. as long as you update the copyright in the app JSON, these will get automatically updated in the next -> main PR

Reply:
Ah, that explains why the integration test only complained about the app JSON file copyright date - I just assumed I needed to update the rest.